prompt-service/src/unstract/prompt_service/helper.py

```diff
@@ -286,8 +300,17 @@ def run_completion(
         )
         highlight_data = None
         if highlight_data_plugin and enable_highlight:
+            fs_instance: FileStorage
+            if execution_source == ExecutionSource.IDE.value:
+                fs_instance = PromptServiceFileHelper.initialize_file_storage(
+                    type=FileStorageType.PERMANENT
+                )
+            if execution_source == ExecutionSource.TOOL.value:
+                fs_instance = PromptServiceFileHelper.initialize_file_storage(
+                    type=FileStorageType.TEMPORARY
+                )
```

Comment:
Is there any case that doesn't come from othere than the IDE or Tool? If not , then make one of this as default.
The `fs_instance: FileStorage` is like not defined, might raise error

Reply:
@muhammad-ali-e Thanks, initializing with Local.